libs/auth/src/injection-tokens/auth-facade.token.ts

```diff
@@ -0,0 +1,3 @@
+import { InjectionToken } from '@angular/core';
+
+export const DaffAuthFacade = new InjectionToken('DaffAuthFacade');
```

Comment:
Do you need this token?

Reply:
It was useful for implementing the mock facade before the facade itself. I thought there was a reason why I had to create the mock facade first but I can't seem to find it now. I'll move the mock facade later and remove the injection token.